ArduCopter/config.h

```diff
@@ -801,3 +801,8 @@
  #define PLANCK_LAND_KP_ALT          .1     // Proportional alt gain used in planck land
  #define PLANCK_NOM_KP_ALT           1
 #endif
+
+// Emergency Mode Throttle setting
+#ifndef PLANCK_HT_THR
+ #define PLANCK_HT_THR              0.6 //60%
```

Comment:
We should change the default to 0.65, to match what we tested. Will probably have to change again after tuning for Indago, but might as well use the best value to date

Reply:
Changed to 0.65